index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+
+</head>
+<body>
+    <header class="header-section">
+        <a href="" class="logo"> My logo </a>
+<nav class="bttn">
+        <span class="line"></span>
+        <span class="line"></span>
+        <span class="line"></span>
+</nav>
```

Comment:
- [ ] Good job adding this section as required :clap: , but I've noticed that your margin does not match what make your design to not stick to the [Figma design](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=the%20toolbar%20(or%20header)%3A) :100:  , Kindly make sure your header stick to the Figma design (e.g. font, colors, images, text, margins, padding)
### Your design
![Screenshot from 2023-01-11 11-06-19](https://user-images.githubusercontent.com/80458199/211766111-efe7c00c-613e-4d68-b7d5-550815580298.png)


### Figma design

![Screenshot from 2023-01-11 11-07-52](https://user-images.githubusercontent.com/80458199/211766180-d5de535a-df6e-4f1c-a495-801674b06ac3.png)